client/src/App.js

```diff
@@ -1,17 +1,28 @@
-import "./App.css";
+// import "./App.css";
 import React from "react";
 import Page from "./pages/Page";
+import GlobalStyle from "./GlobalStyles";
+import styled from "styled-components";
+
+const Main = styled.main`
+  padding: 1rem;
+  max-width: 375px;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+`;
```

Comment:
I would move the styles to the bottom of this file to make it more concise

Reply:
In all the projects I've done before, they were placed on top of the file. 🤷‍♀️